ui/qt-lib/resources/stylesheet.qss

```diff
@@ -316,6 +316,113 @@ QDialog QPushButton[styleRole="passwordVisibilityToggle"]:checked {
     background: #fafafa;
 }
 
+#settingsDialog QComboBox {
+    border: none;
+    background: transparent;
+    padding-right: 5px;
+    color: rgb(52, 52, 62, 127);
+}
+
+#settingsDialog QComboBox::drop-down {
+    border: none;
+    background: transparent;
+  }
+
+#settingsDialog QComboBox::down-arrow {
+    image: url(':/images/expand-arrow.png');
+}
+
+#settingsDialog QComboBox QAbstractItemView {
+    border: 1px solid rgba(0, 0, 0, 30);
+    background: #fafafa;
+}
+
+QPushButton[styleRole="settingsTopbarButton"] {
+    width: 120px;
+    height: 30px;
+    border: none;
+    background: transparent;
+    color: rgb(52, 52, 62, 59);
+}
+
+QPushButton[styleRole="settingsTopbarButton"]:hover {
+    background: rgba(52, 52, 62, 10);
```

Comment:
Again, I don't understand where this opacity value comes from.

Reply:
actually, zeplin doesn't provide any opacity on this button, hover condition isn't specified